app/components/chart-selection-tabs.tsx

```diff
@@ -83,10 +83,9 @@ const useStyles = makeStyles<Theme, { editable: boolean }>((theme) => ({
     padding: `${theme.spacing(1)} ${theme.spacing(3)}`,
     borderRadius: 3,
     transition: "0.125s ease background-color",
-    "&:hover": ({ editable }) => {
-      return editable
-        ? { backgroundColor: theme.palette.grey[200] }
-        : undefined;
+    "&:hover": {
+      backgroundColor: ({ editable }) =>
+        editable ? theme.palette.grey[200] : undefined,
```

Comment:
Ideally this patch should be merged with the previous commit so as not to have a commit that is not working.

Reply:
Good point, thanks!